packages/components/tabs/src/tab-group.spec.ts

```diff
@@ -11,10 +11,18 @@ describe('sl-tab-group', () => {
     beforeEach(async () => {
       el = await fixture(html`<sl-tab-group></sl-tab-group>`);
     });
-  
+
     it('should not break', () => {
       expect(el).shadowDom.to.equalSnapshot();
     });
+
+    it('should have horizontal layout by default', () => {
+      expect(el.getAttribute('vertical')).to.equal('false');
+    });
+
+    it('should have left alignment by default', () => {
+      expect(el.getAttribute('alignment')).to.equal('left');
```

Comment:
`expect(el).to.have.attribute('alignment', 'left');`

Also, perhaps we should use logical values? `start` instead of `left`?

Reply:
I used the same alignment as in Figma

<img width="1157" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/597729e1-c11a-457f-ac6d-334635fca36b">

If we want to change this one I think it also needs to be changed in Figma? @RoaldBoerema what do you think about it? :)

Reply:
Changed (in Figma as well) :)